src/Directory.Build.props

```diff
@@ -19,10 +17,12 @@
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <IsPackable>false</IsPackable>
```

Comment:
What's the rationale for this?

Reply:
Nervousness about deploying a NuGet package for a project just because it is placed in `src`.

Reply:
How often do we have a project in `src` that shouldn't be packaged? (Web projects are not packaged by default.)

What happens if ProjectA is packable and references ProjectB which is not packable? Will it fail, or will it create a ProjectA package with a reference to a non-existent ProjectB?

Reply:
Good questions. Perhaps I'm being unnecessarily paranoid.

Reply:
Okay, dropped the unnecessary references to `IsPackable` and `IsTestProject`. @ddunkin, do you approve?